docs/framework/deployment/repair.md

```diff
@@ -0,0 +1,7 @@
+# Repair the .NET Framework
+
+In some situations, your .NET Framework installation can become damaged and needs to be repaired. This might be the case if your app crashes right after you try to start it of if you cannot install newer .NET Framework versions.
+
+You can repair your .NET Framework install using the [.NET Framework Repair Tool](http://download.microsoft.com/download/2/B/D/2BDE5459-2225-48B8-830C-AE19CAF038F1/NetFxRepairTool.exe). 
+
+If that resolves your issue, that's great. If your app still isn't starting after repairing the .NET Framework, then the problem may be with the application. In that case, you should contact the application publisher.
```

Comment:
I'm not sure that "that's great" doesn't use too friendly a tone. I'd rewrite this to say, "If that doesn't resolve your issue and your app still...."